concord-server/server/src/main/java/com/rabidgremlin/concord/dao/VotesDao.java

```diff
@@ -86,4 +86,7 @@
   @RegisterBeanMapper(GroupedPhraseVoteWithMostRecentVoteTime.class)
   List<GroupedPhraseVoteWithMostRecentVoteTime> getLabelsForUncompletedPhrasesWithMostRecentVoteTime();
 
+  @SqlQuery("SELECT COUNT(*) FROM votes WHERE phraseId = :phraseId")
+  int getVoteCountForPhrase(@Bind("phraseId") String phraseId);
```

Comment:
We'd want to ignore BULK UPLOAD votes here wouldn't we?

Reply:
yes